tests/test_micropub.php

```diff
@@ -908,6 +908,41 @@ function test_create_bookmark() {
 , $post->post_content );
 	}
 
+	function test_create_nested_bookmark() {
```

Comment:
mind adding a comment that this example markup below is generated by Post Kinds? and ideally a link to details in the Post Kind docs.

Reply:
Will note it appropriately. But it could be easily done by a client as within spec.